packages/javascript-sdk/src/oauth2-client/index.ts

```diff
@@ -158,6 +162,12 @@ abstract class OAuth2Client {
     const responseBody = await this.getBody<unknown>(response);
 
     if (response.status !== 200) {
+      const error_description_query_param =
+        new URL(response.url).searchParams.get('error_description') !== undefined;
+
+      if (error_description_query_param) {
+        throw new Error(allowedErrors.PromptError);
+      }
```

Comment:
honestly unsure if this was really a great way to handle this. need to look into this more

Reply:
I don't think this is needed. The `parseError` method used just a few lines down should already grab the error and throw with the right message.